.github/workflows/windows_python.yml

```diff
@@ -1,14 +1,10 @@
 name: windows tests
 
 on:
-  push:
-    branches:
-      - develop
-      - releases/**
-  pull_request:
-    branches:
-      - develop
-      - releases/**
+  workflow_run:
+    workflows: [style and validate]
```

Comment:
I think it would be better to use a name like `style-and-validate` or `style_and_validate` (even if the spaces are allowed in the name, I found it hard to parse this by eye).